posthog/temporal/data_imports/sources/shopify/shopify.py

```diff
@@ -130,35 +130,32 @@ def get_rows():
             yield from _make_paginated_shopify_request(api_url, sess, graphql_object, logger)
             return
 
-        # NOTE: we use inclusive comparisons below because upon testing the shopify APIs with query filters
```

Comment:
don't know what i was on about here... date comparisons are working as expected with > and <.

<img width="1225" height="303" alt="Image" src="https://github.com/user-attachments/assets/e6e58b19-17ef-4cb3-b5b0-0d558b08663c" />